### PR TITLE
Fcu type hybrid variable refrigerant flow unit

### PR DESCRIPTION
**Pull request by kevin-hereworks**
FCU_DSP_VSC_STM_RTM added to FCU.yaml

Hybrid variable refrigerant flow unit with supply and return air temperature monitoring, zone cooling and heating setpoints, fan speed command

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 1]
```diff
@@ -1648,3 +1648,22 @@ FCU_RTC_RHDHS_DTM_DFSS:
   - fire_alarm
   - schedule_run_command
 
+FCU_DSP_VSC_STM_RTM:
+  description: "Hybrid variable refrigerant flow unit with supply and return air temperature monitoring, zone cooling and heating setpoints, fan speed command"
+  implements:
+  - FCU
+  - DSP
+  - VSC
+  - STM
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1657]: unless this FCU serves other equipment, this should be discharge (not supply). please change to `DTM`

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -1648,3 +1648,22 @@ FCU_RTC_RHDHS_DTM_DFSS:
   - fire_alarm
   - schedule_run_command
 
+FCU_DSP_VSC_STM_RTM:
+  description: "Hybrid variable refrigerant flow unit with supply and return air temperature monitoring, zone cooling and heating setpoints, fan speed command"
+  implements:
+  - FCU
+  - DSP
+  - VSC
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1656]: DFVSC? this is for the discharge fan, yes?

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 3]
```diff
@@ -1648,3 +1648,22 @@ FCU_RTC_RHDHS_DTM_DFSS:
...
+  - STM
+  - RTM
+  uses:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1659]: many of these fields are used by the abstract types you've defined. there is no need to redefine them here, so please remove them.

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 4]
```diff
@@ -1648,3 +1648,22 @@ FCU_RTC_RHDHS_DTM_DFSS:
...
+  - run_command
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1660]: `discharge_fan_run_command`?

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 5]
```diff
@@ -1648,3 +1648,22 @@ FCU_RTC_RHDHS_DTM_DFSS:
   - fire_alarm
   - schedule_run_command
 
+FCU_DSP_VSC_STM_RTM:
+  description: "Hybrid variable refrigerant flow unit with supply and return air temperature monitoring, zone cooling and heating setpoints, fan speed command"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1652]: I assume this device has a dx coil and evaporator valve but no data associated with it?

**Review by tasodorff — CHANGES_REQUESTED** [review]
Please see comments and make required adjustments, in particular to replace the supply types with discharge types and to remove the locally defined fields that are already covered by abstract types.

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 1 above)
Comment (kevin-hereworks) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 2 above)
Comment (kevin-hereworks) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 3 above)
Comment (kevin-hereworks) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 5 above)
Comment (kevin-hereworks) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Yes this is correct

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 4 above)
Comment (kevin-hereworks) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: changed

**Comment by cstirdivant** [comment]
@kevin-hereworks please revert this PR to version[cd2bbd8](https://github.com/google/digitalbuildings/pull/912/commits/cd2bbd82f995ea4e3f9bbfa0c4c2af60da9b20f6)

**Comment by kevin-hereworks** [comment]
@cstirdivant should be fixed now

**Pull request merged**